Makefile

```diff
@@ -1,6 +1,6 @@
 RELEASE_VERSION ?= latest
 
-SUBDIRS=kafka-base zookeeper kafka kafka-connect kafka-connect/s2i cluster-controller topic-controller resources
+SUBDIRS=docker-images/kafka-base docker-images/zookeeper docker-images/kafka docker-images/kafka-connect docker-images/kafka-connect/s2i cluster-controller topic-controller resources
```

Comment:
Is it possible to just specify `docker-images` here, and then override `SUBDIRS` in that directory to recurse though the various subdirs?

Reply:
Please have a look at what I did there. It has some slight duplication. But ti allows to build only the docker-images without the controllers, which is IMHO an advantage.

Reply:
Yeah, that looks fine to me.